xml/obs_request_and_review_system.xml

```diff
@@ -101,25 +108,22 @@
    <sect3>
     <title><literal>maintenance_incident</literal></title>
     <para> Official request to open a maintenance incident for official support
-     products. This create by developers who want to start an official
-     maintenance process. Find detail in the maintenance chapter about this. If
-     accepted, a new maintenance incident project is created and package
-     sources listed are copied there. All sources of all actions in one request
-     will be merged into the same maintenance incident. </para>
+     products. These requests are created by developers who want to start an official
+     maintenance process. Details are describe in the maintenance chapter. A new 
+     maintenance incident project is created and package sources get copied there
+     when accepting it. All sources of all actions in one request
+     will be merged into the same maintenance incident project. </para>
    </sect3>
    <sect3>
     <title><literal>maintenance_release</literal></title>
-    <para> Is used to release a ready maintenance update. Unlike
-     maintenance_incident or submit, sources and binaries are copied without a
-     rebuild. Details can be found in maintenance chapter. </para>
+    <para> Is used to release a maintenance update. Unlike sources and binaries are
+     copied without a rebuild. A unique update identifier is also created. 
```

Comment:
Either you need to leave in the explanation about maintenance_incident / submit events or the "Unlike" needs to go:

```suggestion
    <para> Is used to release a maintenance update. Sources and binaries are
     copied without a rebuild. &obs; also creates a unique update identifier.
```